lib/navigation/AppNavigator.tsx

```diff
@@ -12,12 +12,13 @@ import RegisterScreen from "../screens/loginScreens/RegisterScreen";
 import AddNoteScreen from "../screens/AddNoteScreen";
 import EditNote from "../components/EditNote";
 import OnboardingScreen from "../screens/OnboardingScreen";
+import VideoPlayerScreen from "../screens/VideoPlayer"; // Import the VideoPlayerScreen
```

Comment:
Did you create this yourself why not get one from a library?

Reply:
Yes, as tentap does not support inline